src/main/java/com/solodive/championselect/service/dto/riotapi/RiotChampionDTO.java

```diff
@@ -3,19 +3,19 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class Champion {
+public class RiotChampionDTO {
 
     private String version;
     private String id;
     private String key;
```

Comment:
key - pole nie powinno być formatu liczbowego (Long)?

Reply:
Jakbyś zajrzał w tego jsona rito, to byś zobaczył, że wartość jest w cudzysłowie, więc nah.